frontend/apps/picasso/pages/crowdloan-rewards/claim.tsx

```diff
@@ -57,19 +56,49 @@ const ERROR_MESSAGES = {
   },
 };
 
+function getUnvestedAmount(
```

Comment:
Ideally this should be a RPC, but I guess doesn't make sense to spend a lot of time on crowdloan pallet